feat: Add Mastodon, TikTok, and BlueSky social media accounts to links and update assets

YoussefMoRabie opened the pull request:
### What
<!-- description of the PR -->
- Added translations for new social media links (Mastodon, TikTok, and Bluesky)
- Added logo assets for Mastodon, TikTok, and Bluesky
- Updated the 'Contact Us' page with new social media contact links

### Screenshot
<img src="https://github.com/user-attachments/assets/e4d5ed70-b28c-4c75-b88d-84c979457c62" width="400" />

### Part of 
- #6260

YoussefMoRabie committed:
Add translations for new social media links

- Updated .arb files to include translations for Mastodon, TikTok, and Bluesky

YoussefMoRabie committed:
Add assets for Mastodon, TikTok, and Bluesky logos

- Added logo assets for Mastodon, TikTok, and Bluesky to the project

YoussefMoRabie committed:
Update 'Contact Us' page with new social media contacts

- Added Mastodon, TikTok, and Bluesky links to the 'Contact Us' page

monsieurtanuki commented:
@teolemon Just checking: we don't have to populate ALL language files when we add translations, as it's automatically somehow duplicated by the system, right?

monsieurtanuki reviewed (commented):
Hi @YoussefMoRabie!

As coders we don't duplicate translations, we let the system do it from the English translation file.
So, if you add a translated label, do it at least in `app_en.arb` but not beyond - unless you want to check for a specific different language.

That said, the code is good, but I cannot let you imagine that I would be happy to review 126 language files.

So please rollback all your language files except `app_en.arb`.

You also obviously have a problem with the code format (in your case, the indentations).
Configure your IDE so that it automatically formats your code (e.g. when you type Ctrl-S).

YoussefMoRabie committed:
Revert "Add translations for new social media links"

This reverts commit a8c5dfe3e26ff99c1a722c961cf7411c9c216460.

YoussefMoRabie committed:
Add translations for new social media links

YoussefMoRabie committed:
Fix code formatting and indentation issues

YoussefMoRabie commented:
Hi @monsieurtanuki,

Thanks for the feedback! I’ve reverted all language files except app_en.arb and fixed the formatting issues. Sorry for the confusion—this is my first time working with this localization system. I appreciate your patience! Let me know if anything else needs adjustments.

Looking forward to your review! 😊

monsieurtanuki reviewed (approved):
Thank you @YoussefMoRabie!
And congratulations on your first PR here!
